doc/xml/reference.xml

```diff
@@ -481,6 +481,19 @@
                         <example>y</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="repo-default" name="Set Default Repository">
+                        <summary>Set default repository.</summary>
+
+                        <text>Set the default repository for a command to operate on.
+
+                        For example, this option may be used to perform a restore from a specific repository, rather than letting <backrest/> choose.
```

Comment:
That will be more or less true going forward. For example, when doing at restore with a time specified the repos will be searched for the best backup to use, so it might not be clear which one will be selected.

But `rather than using the first configured` is closer to the current usage.